2025-03-27 21:21:18,316 - INFO - Бот успешно инициализирован
2025-03-27 21:21:18,316 - INFO - Бот запущен и готов к работе!
2025-03-27 21:21:35,444 - INFO - Отправлено стартовое сообщение для 5000931101
2025-03-27 21:21:47,479 - INFO - Уведомление отправлено админу 5000931101
2025-03-27 21:21:47,579 - INFO - Тест выполнен для 5000931101
2025-03-27 21:22:20,729 - INFO - Админ 5000931101 начал добавление товара
2025-03-27 21:27:05,494 - ERROR - Infinity polling: polling exited
2025-03-27 21:27:05,495 - ERROR - Break infinity polling
2025-03-27 21:27:35,450 - INFO - === Р‘РѕС‚ СѓСЃРїРµС€РЅРѕ РёРЅРёС†РёР°Р»РёР·РёСЂРѕРІР°РЅ ===
2025-03-27 21:27:35,450 - INFO - === Р‘РѕС‚ Р·Р°РїСѓС‰РµРЅ Рё РіРѕС‚РѕРІ Рє СЂР°Р±РѕС‚Рµ ===
2025-03-27 21:27:57,364 - INFO - РџРѕР»СЊР·РѕРІР°С‚РµР»СЊ 5000931101 РЅР°С‡Р°Р» СЂР°Р±РѕС‚Сѓ СЃ Р±РѕС‚РѕРј
2025-03-27 21:28:03,357 - INFO - РђРґРјРёРЅ 5000931101 РЅР°С‡Р°Р» РґРѕР±Р°РІР»РµРЅРёРµ С‚РѕРІР°СЂР°
2025-03-27 21:28:07,464 - ERROR - РћС€РёР±РєР° РґРѕР±Р°РІР»РµРЅРёСЏ С‚РѕРІР°СЂР°: вќЊ РўСЂРµР±СѓРµС‚СЃСЏ С„РѕС‚Рѕ С‚РѕРІР°СЂР°
Traceback (most recent call last):
  File "C:\Users\РџРљ\Desktop\web app\outfit-lab\api\super.py", line 93, in process_item
    raise ValueError("вќЊ РўСЂРµР±СѓРµС‚СЃСЏ С„РѕС‚Рѕ С‚РѕРІР°СЂР°")
ValueError: вќЊ РўСЂРµР±СѓРµС‚СЃСЏ С„РѕС‚Рѕ С‚РѕРІР°СЂР°
2025-03-27 21:46:00,574 - ERROR - Infinity polling: polling exited
2025-03-27 21:46:00,575 - ERROR - Break infinity polling
2025-03-27 21:46:46,451 - INFO - === Р‘РѕС‚ СѓСЃРїРµС€РЅРѕ РёРЅРёС†РёР°Р»РёР·РёСЂРѕРІР°РЅ ===
2025-03-27 21:46:46,452 - INFO - === Р‘РѕС‚ Р·Р°РїСѓС‰РµРЅ Рё РіРѕС‚РѕРІ Рє СЂР°Р±РѕС‚Рµ ===
2025-03-27 21:47:20,087 - INFO - РџРѕР»СЊР·РѕРІР°С‚РµР»СЊ 5000931101 РЅР°С‡Р°Р» СЂР°Р±РѕС‚Сѓ СЃ Р±РѕС‚РѕРј
2025-03-27 21:59:21,138 - ERROR - Infinity polling: polling exited
2025-03-27 21:59:21,138 - ERROR - Break infinity polling
